androidApp/src/main/res/raw/sample_assessment.json

```diff
@@ -53,6 +53,22 @@
         {"text":"None of the above","exclusive":true}
       ]
     },
+    {
+      "identifier": "checkboxQuestion",
+      "type": "simpleQuestion",
+      "title": "Yes or No?",
+      "subtitle": "Subtitle goes here",
+      "detail": "Some text. This is a test.",
+      "footnote": "This is a footnote.",
+      "image"  : {    "type" : "animated",
+        "imageNames" : ["foo1", "foo2", "foo3", "foo4"],
+        "placementType" : "topBackground",
+        "animationDuration" : 2
+      },
+      "optional": true,
+      "inputItem":{"type" : "checkbox", "identifier": "checkboxQuestionResult","fieldLabel" :  "Check for Yes"},
```

Comment:
Next step: replace the checkbox's placeholder text "Check" with "Check for Yes" from the JSON

Reply:
Next steps
- Refactor CheckboxFragment to use your CheckboxInputView instead of Android's Checkbox
- Open PR

Next step:
- Based on whether the checkbox is selected in the UI, save the state